docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,42 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | True if female                              | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`m_pflichtbeitrag`     | Months of mandatory contributions           | float        |
```

Comment:
Make explanations a little longer (here and below) so as to add information that this is about public pensions and we are talking about months accumulated over working life so far (I guess a "total" in the beginning may do).

Reply:
hmm, I acually did that already in commit [ef39ea3](https://github.com/iza-institute-of-labor-economics/gettsim/pull/396/commits/ef39ea385d222a48c424af98c0f83d0392db0ef2) . seems like this was overwritten somehow.

Reply:
expanded info now